app/.eslintrc.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  root: true,
+  extends: [
+    'nava',
+    // Disable ESLint code formatting rules which conflict with Prettier
```

Comment:
The prettier line does the eslint disable? 🤔

Reply:
Yea, this includes a ruleset that turns off a bunch of eslint rules. Maybe a better way to phrase this code coment?

https://github.com/prettier/eslint-config-prettier/blob/a57a3bb0e5aa39f59c2523bb3b77bf7e13b7be2a/index.js#L23-L33